src/uct/ib/base/ib_iface.c

```diff
@@ -1503,10 +1497,11 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_iface_ops_t *tl_ops,
     self->config.flid_enabled       = config->flid_enabled;
     uct_ib_iface_set_path_mtu(self, config);
 
-    self->config.send_overhead.pre        = config->send_overhead.pre;
-    self->config.send_overhead.pre_bcopy  = config->send_overhead.pre_bcopy;
-    self->config.send_overhead.post       = config->send_overhead.post;
-    self->config.send_overhead.post_zcopy = config->send_overhead.post_zcopy;
+    self->config.send_overhead.bcopy     = config->send_overhead.bcopy;
+    self->config.send_overhead.cqe       = config->send_overhead.cqe;
+    self->config.send_overhead.db        = config->send_overhead.db;
+    self->config.send_overhead.wqe_fetch = config->send_overhead.wqe_fetch;
+    self->config.send_overhead.wqe_post  = config->send_overhead.wqe_post;
```

Comment:
can we assign at once:
self->config.send_overhead = config->send_overhead

Reply:
Fixed.